product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx

```diff
@@ -39,21 +53,29 @@ The restore operation is available for any cluster that has at least one availab
 #### Restore a single-node or primary/standby high-availability cluster
 
 1.  Select the cluster you want to restore on the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com). 
-2.  From **Quick Actions**, select **Restore**.
-3.  On the Restore Cluster page: 
+1.  From **Quick Actions**, select **Restore**.
+1.  On the Restore Cluster page: 
     1.  Fill in the required fields.
-    2.  To restore to the last possible recovery point, in the **Source** section, in the **Point in Time Restore** field, select **Now** on the calendar. Or, to restore further back in time, choose a timestamp. 
-    3.  Review your selections in **Cluster Summary** and select **Restore Cluster** to begin the restore process.  
-4.  The new cluster is now available on the [Clusters](https://portal.biganimal.com/clusters) page.
+    1.  To restore to the last possible recovery point, in the **Source** section, in the **Point in Time Restore** field, select **Now** on the calendar. Or, to restore further back in time, choose a timestamp. 
```

Comment:
Just a nit, I would add where to find the source section and maybe split into two steps: 
```suggestion
    1.  In the **Cluster settings** tab, go to the **Source** section.
    1. To restore to the last possible recovery point, in the **Point in Time Restore** field, select **Now**. Or, to restore further back in time, choose a timestamp. 
```